charts/argo-rollouts/templates/aggregate-roles.yaml

```diff
@@ -37,7 +37,6 @@ rules:
   resources:
   - rollouts
   - rollouts/scale
-  - rollouts/status
```

Comment:
Thank you for your PR.
Since upstream has the permission in latest version, so can you please fix upstream at first ?
*argo-helm follows upstream's manifest.
https://github.com/argoproj/argo-rollouts/blob/v1.6.6/manifests/base/argo-rollouts-aggregate-roles.yaml

Reply:
Hi @yu-croco sure thing, I will open a PR in upstream. Thanks!